diff --git a/node_modules/typescript/lib/tsserver.js b/node_modules/typescript/lib/tsserver.js
index c276781..9cc7d6f 100644
--- a/node_modules/typescript/lib/tsserver.js
+++ b/node_modules/typescript/lib/tsserver.js
@@ -139587,7 +139587,9 @@ var ts;
             "variation",
             "version",
             "virtual",
-            "yields"
+            "yields",
+            "gqlField",
+            "internal"
         ];
         var jsDocTagNameCompletionEntries;
         var jsDocTagCompletionEntries;
diff --git a/node_modules/typescript/lib/typescript.js b/node_modules/typescript/lib/typescript.js
index e8734a6..6db5885 100644
--- a/node_modules/typescript/lib/typescript.js
+++ b/node_modules/typescript/lib/typescript.js
@@ -123451,8 +123451,8 @@ var ts;
         if (configFileName)
             updateExtendedConfigFilesWatches(toPath(configFileName), compilerOptions, watchOptions, ts.WatchType.ExtendedConfigFile);
         return configFileName ?
-            { getCurrentProgram: getCurrentBuilderProgram, getProgram: updateProgram, close: close } :
-            { getCurrentProgram: getCurrentBuilderProgram, getProgram: updateProgram, updateRootFileNames: updateRootFileNames, close: close };
+            { getCurrentProgram: getCurrentBuilderProgram, getProgram: updateProgram, close: close, sourceFilesCache: sourceFilesCache, synchronizeProgram: synchronizeProgram } :
+            { getCurrentProgram: getCurrentBuilderProgram, getProgram: updateProgram, updateRootFileNames: updateRootFileNames, close: close, sourceFilesCache: sourceFilesCache, synchronizeProgram: synchronizeProgram };
         function close() {
             clearInvalidateResolutionsOfFailedLookupLocations();
             resolutionCache.clear();
